### DISCUSSION TITLE
Disable globally set HW_acceleration

### DISCUSSION THREAD

**Discussion by moskovskiy82** [blakeblackshear/frigate#6219] [General]
Without getting into much details (xeonv2+coral+amd hd8570) have 5 cameras working find. Adding another one (reolink B800+neolink in docker) gave me the following problem.
The last camera seems to feed something  hwaccel_args: preset-vaapi doesn't get

`ERROR   : [hevc @ 0x55c0988593c0] No support for codec hevc profile 1.
ERROR   : [hevc @ 0x55c0988593c0] Failed setup for format vaapi: hwaccel initialisation returned error.
ERROR   : Impossible to convert between the formats supported by the filter 'Parsed_fps_0' and the filter 'auto_scale_0'
ERROR   : Error reinitializing filters!
ERROR   : Failed to inject frame into filter network: Function not implemented
ERROR   : Error while processing the decoded data for stream #0:0
ERROR   : main: Unable to read frames from ffmpeg process.
ERROR   : main: ffmpeg process is not running. exiting capture thread...
ERROR   : Ffmpeg process crashed unexpectedly for main.
ERROR   : The following ffmpeg logs include the last 100 lines prior to exit.
ERROR   : [segment @ 0x556f40bc2800] Timestamps are unset in a packet for stream 0. This is deprecated and will stop working in the future. Fix your code to set the timestamps properly
`

So can i just disable hw_accel for this camera? Or maybe more inspiring ideas?

**Comment by NickM-27** [comment]
there's not currently a specific way to do this but you should be able to set it to `" "`
